logging/google/cloud/logging/handlers/transports/background_thread.py

```diff
@@ -263,7 +270,12 @@ def enqueue(self, record, message, resource=None, labels=None,
             'labels': labels,
             'trace': trace,
             'span_id': span_id,
-        })
+        }
+        
+        if self._include_logger_name:
+            log_record['info']['python_logger'] = record.name 
```

Comment:
On line 266 we assign this. Is this if statement needed?

Reply:
Done!